bin/validate-manifests.sh

```
@@ -0,0 +1,68 @@
+# nested maps inside overrides (e.g., { "jsdom": { "tough-cookie": "..." } })
+jq -e '
+  .pnpm?.overrides as $o
+  | if $o == null then empty else
+      ($o | to_entries[] | select(.value|type!="string"))
+      | error("Nested object under pnpm.overrides in " + input_filename)
+    end
+' "$root_pkg" >/dev/null
+
+# 3) duplicate keys inside a single JSON (catches copy/paste errors)
+# jq trick: explode and count repeated keys at the top level & key sub-objects we care about
+dup_keys=$(
```

Shellcheck (0.10.0)

[warning] dup_keys appears unused. Verify use (or export if used externally).

(SC2034)